train.py

```diff
@@ -351,13 +367,11 @@ def main():
         print ('Error processing arguments: (', k, ",", v, ")")
         exit(2)
```

Comment:
As per Guido's main, https://www.artima.com/weblogs/viewpost.jsp?thread=4829, it's better practice to have the function `main()` _return_ its exit status, and have the `if` block that calls main invoke it like: `sys.exit(main())`.
So replace this with `return 2` and replace the call to main, `main()` with `sys.exit(main())`. Or not.